Python/pywarpx/_libwarpx.py

```diff
@@ -561,6 +565,29 @@ def get_particle_arrays(species_name, comp_name, level):
     _libc.free(data)
     return particle_data
 
+def get_particle_boundary_buffer(species_name, boundary, comp_name, level):
```

Comment:
Please add a docstring explaining this function's use and the arguments it takes.